src/utils/daos_hdlr.c

```diff
@@ -1673,9 +1746,11 @@ fs_copy_dir(struct cmd_args_s *ap,
 
 	/* create the destination directory if it does not exist */
 	rc = file_mkdir(ap, dst_file_dfs, dst_path, &tmp_mode_dir);
-	if (rc != EEXIST && rc != 0)
+	if ((rc == EEXIST) && (strncmp(dst_path,"/")!=0)) {
```

Comment:
(style)  space required after that ',' (ctx:VxV)
(style)  spaces required around that '!=' (ctx:VxV)